docs/app/Examples/elements/Button/Groups/ButtonColoredButtonsExample.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react'
+import { Button } from 'stardust'
+
+export default class ButtonColoredButtonsExample extends Component {
+  render() {
+    return (
+      <Button.Group color='blue'>
+        <Button>
+          One
+        </Button>
+        <Button>
+          Two
+        </Button>
+        <Button>
+          Three
+        </Button>
```

Comment:
Sure, we're already expanded I previous examples and conscious about changing you guys stuff!

Sent from my iPhone

> On 27 Jun 2016, at 21:51, Levi Thomason <notifications@github.com> wrote:
> 
> In docs/app/Examples/elements/Button/Groups/ButtonColoredButtonsExample.js:
> 
> > +import React, { Component } from 'react'
> > +import { Button } from 'stardust'
> > +
> > +export default class ButtonColoredButtonsExample extends Component {
> > +  render() {
> > +    return (
> > +      <Button.Group color='blue'>
> > +        <Button>
> > +          One
> > +        </Button>
> > +        <Button>
> > +          Two
> > +        </Button>
> > +        <Button>
> > +          Three
> > +        </Button>
> Can we collapse these solely to save doc site example space?
> 
> <Button>One</Button>
> <Button>Two</Button>
> <Button>Three</Button>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>